src/mdx.ts

```diff
@@ -96,8 +98,8 @@ async function getDocsFrontMatterUncached(): Promise<FrontMatter[]> {
   return frontMatter;
 }
 
-export function getAllFilesFrontMatter(folder: string = 'docs') {
```

Comment:
this `folder` shouldn't be hardcoded as we need to pass it as a parameter for developer docs for example later on

Reply:
Right now it doesn't need to be. We can add it back if needed. We shouldn't add code for the mere possibility of needing it.

Reply:
I just mentioned it since this is deleting something that was already there, but I see what you mean